signer/src/bitcoin/client.rs

```diff
@@ -0,0 +1,59 @@
+//! Bitcoin Core RPC client implementations
+
+use bitcoincore_rpc::RpcApi;
+use sbtc::rpc::BitcoinCoreClient;
+use url::Url;
+
+use crate::{error::Error, keys::PublicKey, util::ApiFallbackClient};
+
+use super::{
+    utxo::{self, SignerUtxo},
+    BitcoinInteract,
+};
+
+/// Implement the [`TryFrom`] trait for a slice of [`Url`]s to allow for a
+/// [`ApiFallbackClient`] to be implicitly created from a list of URLs.
+impl TryFrom<&[Url]> for ApiFallbackClient<BitcoinCoreClient> {
+    type Error = Error;
+    fn try_from(urls: &[Url]) -> Result<Self, Self::Error> {
+        let clients = urls
+            .iter()
+            .map(|url| BitcoinCoreClient::try_from(url.clone()).unwrap())
+            .collect::<Vec<_>>();
+
+        Ok(Self::new(clients))
+    }
+}
+
+impl BitcoinInteract for BitcoinCoreClient {
```

Comment:
Yeah I coulda implemented `BitcoinInteract` for `ApiFallbackClient<BitcoinCoreClient>` and then kept `ApiFallbackClient` out of the `Context` but I didn't... maybe that's the better way to go :thinking:

Reply:
I spoke with Matteo and he seemed to agree with the thoughts that my lazy-angel wanted to ignore (i.e. the "yeah i coulda" above, so I'll make that change.